api/app/models/spree/api/webhooks/payment.rb

```diff
@@ -0,0 +1,12 @@
+module Spree
+  module Api
+    module Webhooks
+      module Payment
+        def void
```

Comment:
@sebastian-palma let's not overwrite methods like this - we can add after_transition hook to any state machine

Reply:
Tried this before and then changed it to keep it working the same way. Payment doesn't have an after `void` transition set, so I had to add an empty method just to be prepended in the api. This way we avoided that.

Reply:
@sebastian-palma let's add that after_transition please, let's avoid prepending

Reply:
Sure, no problem